### PR TITLE
Upgrade flow-go - remove old FVM constructors

### PR DESCRIPTION
**Pull request by janezpodhostnik**
## Description

uses https://github.com/onflow/flow-go/pull/3353
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by sideninja — APPROVED** [review]

**Comment by turbolent** [comment]
@sideninja Was this accidentally merged into a PR branch and not into `master`?

**Comment by sideninja** [comment]
> @sideninja Was this accidentally merged into a PR branch and not into `master`?

I believe I made a mistake in the order of merges. Will make sure it gets resolved. Thank you

**Comment by sideninja** [comment]
@turbolent https://github.com/onflow/flow-emulator/pull/214

File: blockchain.go [hunk 1]
```diff
@@ -1042,11 +1039,13 @@ func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, er
 		nil,
 	)
 
+	r := b.vmCtx.Borrow(env)
+	defer b.vmCtx.Return(r)
```
Comment (turbolent) [inline blockchain.go:1043]: @janezpodhostnik Is this safe? I'm not keeping up with FVM changes, but from what I see, it looks like we keep on using the runtime after the function returned (through the storage that is returned from this function)

Comment (janezpodhostnik) [reply blockchain.go:1043]: Good point. Let me take a look and see if we can make this safer. One option is to simply never return it

**Pull request merged**